jujugui/static/gui/src/app/app.js

```diff
@@ -387,6 +387,8 @@ YUI.add('juju-gui', function(Y) {
       // handlers with a { mask: mask, handlers: handlers } format.
       this.dragNotifications = [];
 
+      this.bakeryFactory = new Y.juju.environments.web.BakeryFactory();
```

Comment:
I'd like to move the bakery into jujulib so this factory can be moved there now.